### ISSUE TITLE
Use the PEP standards for python code

### ISSUE THREAD

**Issue by andarms** [maximinus/pygame-website#11]
i know that you use your own patrons to write code, but python have this [PEPs](https://www.python.org/dev/peps/pep-0008/) standards that help people to easy understand code from others persons and is something about follow the zen of python, the pythonic way to do the things. The major differences between your code and the PEPE are the naming of the variables, you use CamelCase and PEP advised lowercase_with_underscore also you use tabs for the indentation and the advised are 4 spaces. There are also some littles things about the blanks lines.

I dont memorized every rule of PEP i use a highlighter for my text editor Sublime Text to make an easy check of the code. So what do you thing about this? i think that could be beneficial to the project and dont gonna take some much work at this time when the project still in a initial stage.

**Comment by joereynolds** [comment]
I definitely agree with most of what PEP recommends and I think snake-cased variable names should be used.

As for the tabs vs spaces issue, any decent IDE or editor should be able to convert between them so I don't see this as much of an issue.

But yes I agree, the sooner we PEP8-ify the codebase, the better for any future collaborators!

**Comment by maximinus** [comment]
You are correct, I deviate from from pep8 since otherwise function names are the same as variable names. The current standards are:

    ThisIsAClass
    thisIsAFunction
    this_is_a_variable

I don't see any immediate problems because:

1: It's only me doing the code.
2: The code has A standard, just not exactly the same as PEP-8
3: I don't think making it strictly PEP-8 right now would improve the code.
4: Another solution is to just specify in the files the actual differences.
